src/test/java/core/basesyntax/service/FileReaderToolTest.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.exceptions.FileException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderToolTest {
+    private static final String CORRECT_FILE_AND_PATH = "src/test/resources/input/correctInput.csv";
+    private static final String EMPTY_FILE_PATH = "src/test/resources/input/emptyFile.csv";
+    private static final String EMPTY_FILE_EXCEPTION_TEXT = "There is no path to read ";
+    private static final String INCORRECT_FILE_NAME_TO_READ =
+            "src/test/resources/input/incorrectInput.csv";
+    private static final String INCORRECT_FILE_NAME_EXCEPTION_TEXT =
+            String.format("Can't find file \"" + INCORRECT_FILE_NAME_TO_READ + "\" to read ");
+    private static final String INCORRECT_FILE_TYPE_TO_READ = "src/test/resources/correctInput.txt";
+    private static final String INCORRECT_FILE_TYPE_EXCEPTION_TEXT =
+            String.format("Can't find file \"" + INCORRECT_FILE_TYPE_TO_READ + "\" to read ");
+    private static final String INCORRECT_PATH_TO_READ = "src/correctInput.csv";
+    private static final String INCORRECT_PATH_TO_READ_EXCEPTION_TEXT =
+            String.format("Can't find file \"" + INCORRECT_PATH_TO_READ + "\" to read ");
+    private static final FileReaderToolImpl fileReader = new FileReaderToolImpl();
+
+    @Test
+    public void getDataFromCorrectPath_Ok() {
+        List<String> expected = new ArrayList<>();
```

Comment:
better to make test in this way: 1.Precondition 2. Executioin 3. Assertions. And each step separate by empty line